Classes/DDFileLogger.m

```diff
@@ -1031,15 +1031,17 @@ - (DDLogFileInfo *)lt_currentLogFileInfo {
     return _currentLogFileInfo;
 }
 
-- (BOOL)lt_shouldUseLogFile:(nonnull DDLogFileInfo *)logFileInfo {
+- (BOOL)lt_shouldUseLogFile:(nonnull DDLogFileInfo *)logFileInfo isResuming:(BOOL)isResuming {
```

Comment:
Suppose that somebody is using this method ( for some reason ). His implementation will fail due to  unrecognized selector or even worse - he didn't know why his overriding method doesn't change anything. So, his frustration will arise.

Is it intended behavior in hot fix?

I suggest to either forward invocation to this method from old one ( with resuming parameter equal to previous computation scheme of parameter `isResuming` ) or add explicit assertion or NSLogInfo/NSLogWarning to notify developer about new API.

Reply:
`lt_shouldUseLogFile:` was before and is still private API (as are all `lt_`-APIs). If somebody relied on this (which I highly doubt, since it's been in `master` for only a few days), they must expect it to break at any time.